### DISCUSSION TITLE
请问在播放页如何进入歌手页面

### DISCUSSION THREAD

**Discussion by ChengzZg** [SPlayer-Dev/SPlayer#278] [Q&A] [answered]
在播放界面鼠标放在歌手名和专辑名上都会有高光显示，但是点击又没有反应，无法进入歌手页或者专辑页面，需要返回主界面重新搜索，很反直觉，请问是没做这个功能吗

**Comment by imsyy** [comment, marked as answer]
https://github.com/imsyy/SPlayer/actions/runs/11399707184

已修复，还未发版
